reactor-netty-http/src/main/java/reactor/netty/http/client/HttpClient.java

```diff
@@ -1309,25 +1309,28 @@ public RequestSender request(HttpMethod method) {
 	}
 
 	/**
-	 * Specifies the response timeout duration in milliseconds.
-	 * This is time that takes to receive a response after sending a request.
-	 * If the {@code timeout} is {@code null}, any previous setting will be removed and no response timeout
-	 * will be applied.
-	 * If the {@code timeout} is less than {@code 1ms}, then {@code 1ms} will be the response timeout.
-	 * The response timeout setting on {@link HttpClientRequest} level overrides any response timeout
-	 * setting on {@link HttpClient} level.
-	 *
-	 * @param timeout the response timeout duration (resolution: ms)
+	 * Specifies the maximum interval allowed between read operations (resolution: ms).
```

Comment:
This is still a bit unclear to me. Especially, we should clarify what the "read operations" are.

```suggestion
	 * Specifies the maximum duration allowed between each network-level read operation
	 * while reading a given response (resolution: ms).
```

(this also occurs below, making the same suggestion there)

Reply:
I wrapped at 120 characters as stated [here](https://github.com/reactor/reactor-netty/blob/main/CONTRIBUTING.md). Let me know if that's changed.